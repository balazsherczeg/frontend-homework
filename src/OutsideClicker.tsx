import DropdownCloser from './DropdownCloser';

const OutsideClicker = () => (
  <DropdownCloser>
    <div className="OutsideClicker" />
  </DropdownCloser>
)

export default OutsideClicker