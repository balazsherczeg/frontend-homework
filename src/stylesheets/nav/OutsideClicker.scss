.OutsideClicker {
  @include overlay;

  background: transparent;
}
