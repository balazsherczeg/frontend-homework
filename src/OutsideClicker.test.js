import { shallow } from 'enzyme'
import OutsideClicker from './OutsideClicker'

describe('<OutsideClicker />', () => {
  it('renders', () => {
    expect(shallow(<OutsideClicker />)).toMatchSnapshot()
  })
})
