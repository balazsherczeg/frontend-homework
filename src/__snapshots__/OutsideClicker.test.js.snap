// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<OutsideClicker /> renders 1`] = `
<Connect(DropdownCloser)>
  <div
    className="OutsideClicker"
  />
</Connect(DropdownCloser)>
`;
